config.json

```diff
@@ -432,7 +432,11 @@
       "core": false,
       "difficulty": 5,
       "slug": "run-length-encoding",
-      "topics": null,
+      "topics": [
+        "strings",
+        "integers",
+        "sequences"
```

Comment:
Great suggestions :) I agree with `strings` and `integers` but I'm not sure about `sequences`. How do you think that relates to the exercise?

I would also maybe add `transforming` and maybe `pattern_matching`? What do you think? :)

Reply:
good catch, I will add these changes 👍